primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-es.js

```diff
@@ -144,68 +144,68 @@ if (window.PrimeFaces) {
       "javax.faces.converter.DateTimeConverter.DATETIME_detail": "{2}: '{0}' no se ha podido reconocer como fecha y hora. Ejemplo: {1}",
-      "javax.faces.converter.DateTimeConverter.PATTERN_TYPE": "{1}: debe especificarse el atributo 'pattern' o 'type' para convertir el valor '{0}'",
+      "javax.faces.converter.DateTimeConverter.PATTERN_TYPE": "{1}: debe especificarse el atributo 'patrón' o 'tipo' para convertir el valor '{0}'",
       "javax.faces.converter.NumberConverter.CURRENCY": "{2}: '{0}' no se ha podido reconocer como un valor de divisa.",
       "javax.faces.converter.NumberConverter.CURRENCY_detail": "{2}: '{0}' no se ha podido reconocer como un valor de divisa. Ejemplo: {1}",
       "javax.faces.converter.NumberConverter.PERCENT": "{2}: '{0}' no se ha podido reconocer como porcentaje.",
       "javax.faces.converter.NumberConverter.PERCENT_detail": "{2}: '{0}' no se ha podido reconocer como porcentaje. Ejemplo: {1}",
-      "javax.faces.converter.NumberConverter.NUMBER": "{2}: '{0}' no se ha podido reconocer como fecha.",
+      "javax.faces.converter.NumberConverter.NUMBER": "{2}: '{0}' no es un número.",
       "javax.faces.converter.NumberConverter.NUMBER_detail": "{2}: '{0}' no es un número. Ejemplo: {1}",
       "javax.faces.converter.NumberConverter.PATTERN": "{2}: '{0}' no es un patrón numérico.",
       "javax.faces.converter.NumberConverter.PATTERN_detail": "{2}: '{0}' no es un patrón numérico. Ejemplo: {1}",
-      "javax.faces.validator.LengthValidator.MINIMUM": "{1}: Error de validación: el largo es inferior que el mínimo permitido de '{0}'",
-      "javax.faces.validator.LengthValidator.MAXIMUM": "{1}: Error de validación: el largo es mayor que el máximo permitido de '{0}'",
-      "javax.faces.validator.RegexValidator.PATTERN_NOT_SET": "Se debe establecer el patrón de Regex.",
-      "javax.faces.validator.RegexValidator.PATTERN_NOT_SET_detail": "El patrón Regex debe establecerse en un valor no vacío.",
-      "javax.faces.validator.RegexValidator.NOT_MATCHED": "El patrón Regex no coincide",
-      "javax.faces.validator.RegexValidator.NOT_MATCHED_detail": "El patrón Regex de '{0}' no coincide",
-      "javax.faces.validator.RegexValidator.MATCH_EXCEPTION": "Error en expresión regular.",
-      "javax.faces.validator.RegexValidator.MATCH_EXCEPTION_detail": "Error en expresión regular, '{0}'",
+      "javax.faces.validator.LengthValidator.MINIMUM": "{1}: Error de validación: La longitud es inferior al mínimo permitido de '{0}'",
+      "javax.faces.validator.LengthValidator.MAXIMUM": "{1}: Error de validación: La longitud es superior al máximo permitido de '{0}'",
+      "javax.faces.validator.RegexValidator.PATTERN_NOT_SET": "El patrón de la expresión regular no debe estar vacío.",
+      "javax.faces.validator.RegexValidator.PATTERN_NOT_SET_detail": "El patrón de la expresión regular no debe estar vacío.",
```

Comment:
"El patrón de la expresión regular no puede estar vacío.", IMO

Reply:
Yes, It's better to use `puede` than `debe`.